pkg/resmgr/control/cpu/cpu.go

```diff
@@ -125,16 +124,11 @@ func (ctl *cpuctl) PostStopHook(c cache.Container) error {
 
 // enforceCpufreqGovernor enforces a class-specific cpufreq governor to a cpuset.
 func (ctl *cpuctl) enforceCpufreqGovernor(class string, cpusIds ...int) error {
-	if _, ok := ctl.classes[class]; !ok {
-		return fmt.Errorf("non-existent cpu class %q", class)
-	}
-	governor := ctl.classes[class].FreqGovernor
-	for cpu := range cpusIds {
-		log.Info(strconv.Itoa(cpu), governor)
-	}
-	log.Debug("enforcing cpu frequency governor %q on %v", governor, cpusIds)
-	if err := utils.SetScalingGovernorForCPUs(cpusIds, governor); err != nil {
-		return fmt.Errorf("Cannot set cpufreq governor %d: %w", governor, err)
+	if governor := ctl.classes[class].FreqGovernor; governor != "" {
+		log.Debug("enforcing cpu frequency governor %q from class %q on %v", governor, cpusIds)
```

Comment:
The log message uses a format string with 3 %verbs but only 2 arguments.

Reply:
good catch. Fixed.